usr.sbin/daemon/daemon.c

```diff
@@ -49,12 +48,14 @@
 #include <stdlib.h>
 #include <unistd.h>
 #include <string.h>
-#include <strings.h>
 #define SYSLOG_NAMES
 #include <syslog.h>
 #include <time.h>
 #include <assert.h>
 
+/* 1 year in seconds */
+#define MAX_RESTART_DELAY 31536000
```

Comment:
I'd prefer spelling this out as `60 * 60 * 24 * 365` to make it also obvious that it's correct in the definition itself